web/thesauruses/lua/5/strings.json

```
@@ -0,0 +1,272 @@
+{
+    "meta": {
+      "language": "lua",
+      "language_name": "Lua",
+      "structure": "strings",
+      "language_version": "5.3.6"
+    },
+    "concepts": {
+      "is_primitive_or_not": {
+        "name": "Is this a built-in type in this language?",
+        "code": "Yes"
+      },
+      "import": {
+        "name": "Import the string class",
+        "code": "",
+        "comment": "Does not require to import as it is in builtins"
+      },
+      "default_string_byte_encoding": {
+        "name": "Default byte encoding (ex: ASCII UTF-8, UTF-16, etc.)",
+        "not-implemented": true
+      },
+      "create_new_string": {
+        "name": "Create new string",
+        "code": "str1 = 'New string Example1' \nstr2 = \"New string Exampl2\""
+      },
+  
+      "create_multiline_string": {
+        "name": "Create new multi-line string",
+        "code": "[['Multiline string']]"
+      },
+      "assign_new_string": {
+        "name": "Assign string from another string",
+        "code": "a = 'a'\na = 'b'"
+      },
+      "destroy_string": {
+        "name": "Destroy string",
+        "not-implemented": "true"
+      },
+      "length_of_string": {
+        "name": "Length of string",
+        "code": "string.len('string')"
+      },
+      "max_length_of_string": {
+        "name": "Maximum length of string",
+        "code": "230,000 characters"
+      },
+      "clear_string": {
+        "name": "Clear/empty string",
+        "code": "a = 'a'\na=''"
+      },
+      "is_empty": {
+        "name": "Is string empty?",
+        "code": "str = ''\nif (str == nil or str == '') then\nprint('str is empty')\nend"

```

Can you inden't the code inside the `if`, also there is a more readable way to put multiple lines like so:
```suggestion
        "code": [
            "str = ''",
            "if (str == nil or str == '') then",
            "    print('str is empty')",
            "end"
        ]
```

---

I keep it without  indent because in lau we do not need to specify indent.